clusters/previewnet/common/helmrelease.yaml

```
@@ -0,0 +1,59 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: mirror
+  namespace: common
+spec:
+  chart:
+    spec:
+      chart: hedera-mirror-common
+      sourceRef:
+        kind: HelmRepository
+        name: hedera-mirror-node
+      version: '0.60.0'
```

Since it has its own cluster now we should use the same versioning strategy:
```suggestion
      version: '>0.0.0-0' # Update to the latest pre-release
```

---

Fixed!